ci-operator/step-registry/hypershift/mce/ibmz/openshift-extended-test-s390x/hypershift-mce-ibmz-openshift-extended-test-s390x-commands.sh

```
@@ -0,0 +1,655 @@
+            echo "test additional: ${test_additional:1}"
+            test_scenarios="${test_scenarios}|${test_additional:1}"
+        else
+            echo "test additional: ${test_additional:1:-1}"
+            test_scenarios="${test_scenarios}|${test_additional:1:-1}"
+        fi
+    fi
+
+    echo "final scenarios: ${test_scenarios}"
+    echo "SHARD_ARGS=\"${SHARD_ARGS}\""
+
+    extended-platform-tests run all --dry-run ${SHARD_ARGS:-} | \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)